src/index.js

```diff
@@ -0,0 +1,83 @@
+import './style.css';
+import { startGame } from './components/startGame';
+import { checkWin } from './components/checkWin';
+import { resetGame } from './components/resetGame';
+import { checkDraw } from './components/checkDraw';
+import { markPosition } from './components/markPosition';
+
+const firstPlayerNameInput = document.querySelector('.first-player-input');
+const secondPlayerNameInput = document.querySelector('.second-player-input');
+
+const playerOClass = 'circle';
+const playerXClass = 'cross';
+
+const startGameButton = document.querySelector('.start-game-button');
+const startWrapper = document.querySelector('.start-wrapper');
+const currentTurnWrapper = document.querySelector('.current-turn');
+
+const board = document.querySelector('.board');
+const cells = document.querySelectorAll('.cell');
+const bottomControls = document.querySelector('.winning-message-wrapper');
+const winningMessage = document.querySelector('.winning-message');
+const restartButton = document.querySelector('.restart-button');
+let isFirstPlayerTurn = true;
+
+let gameState = [
```

Comment:
`let` is not necessary here

Reply:
to chyba musi być 'let', bo bez tego nie mogę "wyczyścić" tablicy z wyników poprzedniej gry po wciśnięciu resetu

Reply:
We could deal with it by doing
```
gameState[0] = [null, null, null];
gameState[1] = [null, null, null];
gameState[2] = [null, null, null];
```
instead of 
```
gameState = [
    [null, null, null],
    [null, null, null],
    [null, null, null],
];
```
Then, `gameState` can be declared using `const`